src/test/java/net/snowflake/client/jdbc/DatabaseMetaDataInternalLatestIT.java

```diff
@@ -37,6 +37,36 @@ public void tearDown() throws Exception {
     }
   }
 
+  @Test
```

Comment:
Add test to assert that when CLIENT_METADATA_REQUEST_USE_CONNECTION_CTX=true and the schema parameter is null and the session schema contains a wildcard character, the show command is scoped to the schema level (not database level like previously).